S21/hvkone/ui_txt_conv.py

```diff
@@ -0,0 +1,81 @@
+#---------------------------------
+# DEPENDECIES
+#---------------------------------
+# pip install langdetect
+from langdetect import detect
+
+# pip install translate
+from translate import Translator
+
+# pip install pycountr
+import pycountry
+
+# pip install bcp47
+from bcp47 import bcp47
+
+# pip install langcodes
+# pip install language_data
+from langcodes import *
```

Comment:
This again needs a library to install first to run the code. For us, it does not look like a good idea to install so many libraries because the code reviewer has to install all these libraries to run and test the code. Is it possible if we can change the code and run it without these libraries?